app/scripts/controllers/transactions/index.js

```diff
@@ -2662,4 +2665,20 @@ export default class TransactionController extends EventEmitter {
   _getApprovalId(txMeta) {
     return String(txMeta.id);
   }
+
+  /**
+   * Creates approvals for all unapproved transactions in the txStateManager.
+   *
+   * @returns {Promise<void>}
+   */
+  async initApprovals() {
```

Comment:
I think the standard is all public methods then private so we can move this method up under the last public method.

Reply:
Fixed